specs/_features/whisk/fork.md

```diff
@@ -127,12 +123,13 @@ def upgrade_to_whisk(pre: bellatrix.BeaconState) -> BeaconState:
 
     # Do a candidate selection followed by a proposer selection so that we have proposers for the upcoming day
     # Use an old epoch when selecting candidates so that we don't get the same seed as in the next candidate selection
-    whisk_candidate_selection(post, epoch - WHISK_PROPOSER_SELECTION_GAP - 1)
-    whisk_proposer_selection(post, epoch)
+    initial_candidate_gap = WHISK_PROPOSER_SELECTION_GAP + Epoch(1)
+    select_whisk_candidate_trackers(post, max(epoch, initial_candidate_gap) - initial_candidate_gap)
```

Comment:
The max() here serves as underflow protection, right?

Reply:
Yes! should I add a comment?

Reply:
Sure. Thanks.